net/seafile-server/Makefile

```diff
@@ -72,7 +72,9 @@ endif
 
 PKG_BUILD_DEPENDS:=vala/host libevhtp
 
-TARGET_CFLAGS += -minterlink-mips16
+ifdef CONFIG_USE_MIPS16
+	TARGET_CFLAGS += -minterlink-mips16
+endif
```

Comment:
I know that but something overrides that setting and appends the flag "-mnointerlink-mips16" to the list of CFLAGs. It likely comes from the pkgconfig of a dependency or maybe a default build setting for the sources, will try to find out and commit a "proper" fix soon.